src/react-components/ui-root.js

```diff
@@ -1415,7 +1426,7 @@ class UIRoot extends Component {
                         }}
                       />
                     )}
-                    {this.state.sidebarId !== "chat" && this.props.hub && (
+                    {!isLockedDownDemo && this.state.sidebarId !== "chat" && this.props.hub && (
```

Comment:
Looking at this again, as we don't really want any messages, I think we should be good just remoing this tow `isLockDownDemo` from:
https://github.com/mozilla/hubs/blob/a87f4b937fb4fa3b8badf4f116eb8489b329d72d/src/react-components/ui-root.js#L1429
and
https://github.com/mozilla/hubs/blob/a87f4b937fb4fa3b8badf4f116eb8489b329d72d/src/react-components/ui-root.js#L1453

and move it to where we receive the messages:
https://github.com/mozilla/hubs/blob/a87f4b937fb4fa3b8badf4f116eb8489b329d72d/src/message-dispatch.js#L58-L61

We can just return there if it's a lock down demo:
```javascript
  receive(message) {
    if (isLockedDownDemoRoom()) return;

    this.addToPresenceLog(message);
    this.dispatchEvent(new CustomEvent("message", { detail: message }));
  }
```
so we minimize the changes surface.

Reply:
Agreed! This is a more elegant solution!